samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -352,6 +353,7 @@ static void uart_rx_handler(u8_t character)
 	return;
 send:
 	uart_rx_disable(uart_dev);
+	k_sleep(100);
```

Comment:
it's interrupt context. Cannot call k_sleep here.

Reply:
Fixed